src/components/PosterGrid/PosterGrid.js

```diff
@@ -2,24 +2,28 @@ import React, { Component } from 'react'
 import Poster from '../Poster/Poster'
 import Movie from '../Movie/Movie'
 import './PosterGrid.css'
+import fetchData from '../../apiCalls'
 
 class PosterGrid extends Component {
   constructor(props) {
     super(props);
     this.state = {
       mainView: true,
-      currentId: null,
+      movieData: null,
```

Comment:
Rather than null, it's a best practice to set the initial state with the data type it will eventually become - probably an empty object in this case.